spec/std/uri_spec.cr

```diff
@@ -54,6 +54,18 @@ describe "URI" do
     end
   end
 
+  describe ".absolute?" do
+    it { URI.parse("http://www.example.com/foo").absolute?.should eq(true) }
+    it { URI.parse("http://www.example.com").absolute?.should eq(true) }
+    it { URI.parse("/foo").absolute?.should eq(false) }
+  end
+
+  describe ".relative?" do
```

Comment:
I wouldn't add extensive specs for `relative?` since it is just `! absolute?`. One example to ensure it generally works is enough.
The other examples should be moved to spec `absolute?` to spec all variants in one place.